external/CMakeLists.txt

```diff
@@ -66,6 +66,36 @@ add_subdirectory(db_drivers)
 add_subdirectory(randomx EXCLUDE_FROM_ALL)
 add_subdirectory(date EXCLUDE_FROM_ALL)
 
+install(
+  DIRECTORY boost
+  DESTINATION include${OXEN_INSTALL_INCLUDEDIR_SUFFIX}
+  COMPONENT   development
+  FILES_MATCHING PATTERN "*.hpp")
+
+install(
+  DIRECTORY oxen-logging/include/oxen
+  DESTINATION include${OXEN_INSTALL_INCLUDEDIR_SUFFIX}
+  COMPONENT   development
+  FILES_MATCHING PATTERN "*.hpp")
+
+install(
+  DIRECTORY oxen-logging/spdlog/include/spdlog
+  DESTINATION include${OXEN_INSTALL_INCLUDEDIR_SUFFIX}
+  COMPONENT   development
+  FILES_MATCHING PATTERN "*.h")
+
+install(
+  DIRECTORY oxen-logging/fmt/include/fmt
+  DESTINATION include${OXEN_INSTALL_INCLUDEDIR_SUFFIX}
+  COMPONENT   development
+  FILES_MATCHING PATTERN "*.h")
+
+install(
+  DIRECTORY nlohmann-json/include/nlohmann
+  DESTINATION include${OXEN_INSTALL_INCLUDEDIR_SUFFIX}
+  COMPONENT   development
+  FILES_MATCHING PATTERN "*.hpp")
```

Comment:
Are these submodule header installs needed?  They are highly likely to conflict with existing headers of the actual packages on the system, which seems undesirable.